application/gqlgen/schema.graphql

```diff
@@ -114,24 +114,21 @@ type Mutation {
     createOrganizationTrust(input: CreateOrganizationTrustInput!): Organization!
     removeOrganizationTrust(input: RemoveOrganizationTrustInput!): Organization!
 
-    """
-    NOT YET IMPLEMENTED --
-    Bulk create `MeetingInvitation`s
-    """
-    createMeetingInvitations(input: CreateMeetingInvitationsInput!): [MeetingInvitation!]!
+    "Bulk create `MeetingInvite`s and return the updated list of invites for the specified meeting."
```

Comment:
Should we add something like ... `can be called multiple times with repeat email addresses being ignored.`?